PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -208,7 +212,7 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
 
   /// Collision selection
   /// \param collision  collision table with
-  template <bool applyTrigSel, bool applyCentSel, typename Col>
+  template <bool applyTrigSel = false, int centEstimator = CentralityEstimators::None, typename Col>
```

Comment:
```suggestion
  template <bool applyTrigSel, int centEstimator, typename Col>
```
Following up on my previous comment, I would suggest to always require to explicitly provide all arguments for each instance.

Reply:
Sorry, I understood the opposite :)